README.md

```diff
@@ -20,6 +20,8 @@ The Provider is designed for benefits like:
 
 ## Usage
 
+See [this guide](https://registry.terraform.io/providers/iterative/iterative/latest/docs/guides/getting-started) for more information.
```

Comment:
That's a 404.

Reply:
Why not link to our own Get Started though? Or some section on TPI (and link from there to Terraform docs)

Reply:
it won't be a 404 after this PR